src/adapter/src/session/vars.rs

```diff
@@ -1141,6 +1153,9 @@ pub struct SystemVars {
     // features
     enable_with_mutually_recursive: SystemVar<bool>,
 
+    // storage configuration
+    enable_multi_worker_storage_persist_sink: SystemVar<bool>,
```

Comment:
If you don't want this to be accidentally overridden by a LD parameter you should also add the following line to `SystemVars::iter_synced()`:

```rust
.filter(|v| v.name() != ENABLE_MULTI_WORKER_STORAGE_PERSIST_SINK.name)
```

Reply:
I wan this to be controllable with and LD parameter tho!

Reply:
Sorry, than ignore this.